servicepulse/intro-pending-retries.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Pending Retries Message Management in ServicePulse
+summary: Describes how ServicePulse monitors archived failed messages, and allows unarchiving archived failed messages.
+tags:
+ - ServicePulse
+reviewed: 2016-07-28
+---
+
+
+In Versions **1.6.6** and above ServicePulse includes a screen to view and manage failed messages that have been sent for retry but are pending final status.
+
+Pending retries messages can be found by going to the pending retries screen.
+
+![Pending Retries Tab](images/pending-retries.png 'width=500')
+
+
+### Pending Retries Messages
+
+The Pending Retries Messages screen shows failed messages which have been retried but the status of that retry is pending. The status of retried failures is updated when either the message is processed again as either an Audited message (success) or as a failed message.
+
+Failed messages that are retried may stay in the pending state for the following reasons:
+ - The retrying endpoint is not working (e.g. crashed or is scaled-out) and the retried messages is waiting in the queue and has not yet been processed.
+ - The retrying endpoint does not have auditing enabled but has successfully processed the retried message.
```

Comment:
With regards to scale-out what you say is true if one is using competing consumers, than at lease 1 consumer makes the trick. However if one is using distributor or Sender Side Distribution like here:
http://docs.particular.net/nservicebus/msmq/scalability-and-ha/sender-side-distribution#mapping-physical-endpoint-instances
So basically for MSMQ scale-out, than you need to move the message from 1 queue to another. Having said that what if we change your text to something like:

If the endpoint where the retried message is sent to is not running, e.g. crashed or scaled-out using [Sender-side distribution](/nservicebus/msmq/scalability-and-ha/sender-side-distribution), then the retried messages would be waiting in the queue.